src/afl-fuzz-cmplog.c

```diff
@@ -296,7 +298,9 @@ void init_cmplog_forkserver(afl_state_t *afl) {
            "options\n"
            "      fail, poke <afl-users@googlegroups.com> for troubleshooting "
            "tips.\n",
-           DMS(afl->fsrv.mem_limit << 20), afl->fsrv.mem_limit - 1);
+           stringify_mem_size(val_buf, sizeof(val_buf),
```

Comment:
CmpLog needs -m none (it maps sever megabytes of shared mem). Should we handle it in a specific way?